### ISSUE TITLE
[SP-API] Fulfillment Inbound API v2024-03-20 createInboundPlan ERROR: Prep classification for this SKU was missing.

### ISSUE THREAD

**Issue by littleylv** [amzn/selling-partner-api-docs#3708]
## I'm submitting a...
<pre><code>[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## SP-API Name and Version or Report/Feed Type
https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference#createinboundplan
Fulfillment Inbound API v2024-03-20 createInboundPlan

## Expected Behavior
[ItemInput](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference#iteminput) should have a property like [the v0 version InboundShipmentPlanRequestItem](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference#iteminput) PrepDetailsList so we can pass the propType property.

## Current Behavior
API returns error:
```
[severity] => ERROR
[code] => FBA_INB_0182
[details] => There's an input error with the resource '{MY_SKU}'.
[message] => ERROR: Prep classification for this SKU was missing. Choose the prep category that applies to this SKU and apply any required prep and labeling to each sellable unit.
```

**Comment by alexartisans** [comment]
The same issue. No way to specify this data. 
![image](https://github.com/amzn/selling-partner-api-docs/assets/103959517/4e83efe4-5f1e-4999-9db2-b30a6aec7bbf)
This is where its reflected on the front end. The whole API is unusable at this time...

**Comment by zhihui** [comment]
I encountered the same issue.

**Comment by johnkw** [comment]
Also prevented from creating any shipments due to this.  We've had a Seller Support ticket open since 2024-04-02 and it's still pending an answer.

**Comment by homes2001** [comment]
same issue for us

**Comment by marintudela** [comment]
Same here, getting the FBA_INB_0182 error even when I sent the info in the json

**Comment by MeenaAmz** [comment]
To handle this error, please add the prep category information for SKUs via Send to Amazon UI before proceeding with the CreateInboundPlan request. This has also been noted in the [usecase guide](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-use-case-guide#step-1-create-an-inbound-plan). This is a one time activity that needs to be done for a new listing. We will be looking into making this more seamless via APIs in the future.

**Comment by alexartisans** [comment]
We have 42,000 SKUs. Are you proposing we do that to all of them???? Also How would we get prep requirements ??????? No call in new API !!!! What's a POINT of API IF you have to use "UI" for the API to work ???????

**Comment by johnkw** [comment]
Ok so the answer is it's currently broken/missing from the API.  But, roughly what's the timeline on getting this fixed in the API itself, as opposed to a hack workaround of manually clicking in SellerCentral?  Days?  Months?

**Comment by littleylv** [comment]
> To handle this error, please add the prep category information for SKUs via Send to Amazon UI before proceeding with the CreateInboundPlan request. This has also been noted in the [usecase guide](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-use-case-guide#step-1-create-an-inbound-plan). This is a one time activity that needs to be done for a new listing. We will be looking into making this more seamless via APIs in the future.

We have thousands of SKUs. Handling that through UI is a bad choice.
You should make it possible through API before it's fully launched.

**Comment by johnkw** [comment]
After 4 weeks of having a ticket open, SellerSupport finally confirmed that there is no way to get past this error in the API, and we need to wait for the API to implement this (or use the crazy hack workaround of massive manual clicking around in SellerCentral). They will not provide a timeline though.

**Comment by garply1** [comment]
Any word on the timeframe on this? We have 400,000+ skus. This is unacceptable.

**Comment by alexartisans** [comment]
Nope... They feel its not important because v0 is still "working"... Wish they rolled v03-20 out 6 months ago so they had time to address all the bugs... Someone can write a chrome extension that does it... I would if Amazon gave us any ETA or if they told us they wont address it...
